### PR TITLE
add 88 merge-sorted-array

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 20m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 88.merge-sorted-array.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
+        """
+        Do not return anything, modify nums1 in-place instead.
+        """
+        """
+        time: O((n+m)log(n+m))
+        space: O(1)  if (len(nums1) - m) > n
+             : O(n)  else
+        """
+        del nums1[m:]
```
Comment (ryuji0123) [inline 88.merge-sorted-array.py:18]: It seems to be weird to delete elements to modify elements in-place

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
```py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        for i, n in enumerate(sorted(nums1[: m] + nums2)):
            nums1[i] = n
```

**Comment by kobakobashu** [comment]
I fixed my code.  Please check it.

**Review by ryuji0123 — APPROVED** [review]

**Pull request merged**